modules/ossm-troubleshooting-operators.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-troubleshooting-operators_{context}"]
+= Troubleshooting service mesh Operators
+
+If you experience Operator issues:
+
+* Verify your Operator subscription status.
+* Verify that you did not install a community version of the Operator, instead of the supported Red Hat version.
+* You can install Operators only through the OpenShift console, the OperatorHub is not accessible from the command line.
+* You must have the `cluster-admin` role to install {ProductName}.
+* Check for any errors in the Operator pod logs if the issue is related to installation of Operators.
```

Comment:
```suggestion
* Verify that you have the `cluster-admin` role to install {ProductName}.
* Check for any errors in the Operator pod logs if the issue is related to installation of Operators.

[NOTE]
====
You can only install Operators through the OperatorHub in the OpenShift web console. The OperatorHub is not accessible from the command line.
====
```

Suggestion - I'd make the one that seems out of place in the list a note instead and make the others consistent with "verify..."